src/plugins/videojs-http-source-selector/plugin.js

```diff
@@ -46,9 +46,13 @@ const onPlayerReady = (player) => {
     } else {
       console.log('player.videojs_http_source_selector_initialized == false');
       player.videojs_http_source_selector_initialized = true;
-      const controlBar = player.controlBar,
-        fullscreenToggle = controlBar.getChild('fullscreenToggle').el();
-      controlBar.el().insertBefore(controlBar.addChild('SourceMenuButton').el(), fullscreenToggle);
+      const controlBar = player.controlBar;
+      const fullscreenToggle = controlBar.getChild('fullscreenToggle');
+      if (fullscreenToggle) {
+        controlBar.el().insertBefore(controlBar.addChild('SourceMenuButton').el(), fullscreenToggle.el());
+      } else {
+        controlBar.el().appendChild(controlBar.addChild('SourceMenuButton').el());
+      }
```

Comment:
This also triggered an error in our custom components page since `seekThumbnails` assumes there's a progress bar (which makes sense), and another one since `http-source-selector` assumes there's a `fullscreenToggle`

Reply:
ok but i do not see you are checking here whether seekThumbnails is enabled or not?

Reply:
yeah so we had two issues on that page - the first one was because this example removes the progress-bar and `seekThumbnails` assumes it exists, fixed by disabling seekThumbs in this example. The second one then showed up because the quality-selector assumes the full-screen toggle exists, which is also wrong (but not directly related to seek-thumbnails)